src/Components/TrainingListPage.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text, ImageBackground, TouchableOpacity, FlatList } from 'react-native';
+import { connect } from 'react-redux';
+import theme from '../styles/theme.style';
+import * as list from '../styles/list.style';
+
+const getTrainingDuration = trainingDrills => {
+  const durationList = trainingDrills.map(({ durationInMinutes }) => durationInMinutes);
+  return durationList.reduce((a, b) => a + b, 0);
+};
+
+const getTrainingNbPlayers = trainingDrills => {
+  const nbPlayersList = trainingDrills.map(({ nbPlayers }) => nbPlayers);
+  return Math.max(...nbPlayersList);
+};
+
+const mapStateToProps = state => {
+  return {
+    trainings: state.trainings,
+    allDrills: state.drills,
+  };
+};
+
+export const TrainingListPage = props => {
+  const { allDrills, trainings, navigation } = props;
+
+  const [data] = useState(trainings);
+
+  return (
+    <View style={styles.trainingListPage}>
```

Comment:
You should use destructuration for `styles` and `item` as well :)
It's easier like this to know what the component is using

Reply:
as disscussed together, will do in #26